crates/iceberg-ext/src/catalog/rest/error.rs

```diff
@@ -234,6 +235,30 @@ impl ErrorModel {
     }
 }
 
+#[derive(Debug, Serialize, Deserialize, ToSchema)]
+pub struct ApiErrorModel {
```

Comment:
I am wondering if we should ditch the `iceberg-ext` `ErrorModel` completely eventually.

Reply:
As in move it into iceberg-catalog?